### PR TITLE
Added MYST Token to Defaults

### PR DESCRIPTION
**Pull request by PhABC**
Website : https://mysterium.network/
Slack : http://slack.mysterium.network/

### CODE REVIEW

File: app/scripts/tokens/ethTokens.json [hunk 1]
```diff
@@ -198,6 +198,12 @@
     "type":"default"
   },
   {
+    "address":"0xa645264c5603e96c3b0b078cdab68733794b0a71 ",
```
Comment (l0c0luke) [inline app/scripts/tokens/ethTokens.json]: space at the end matters?

**Comment by PhABC** [comment]
Good catch, shouldn't be there.

**Comment by PhABC** [comment]
Edited!

**Comment by tayvano** [comment]
You're support is either epic or your users are smart. We haven't had many tickets at all about your token itself. I added your links to the docs we have. Thank you very much!

**Comment by PhABC** [comment]
Haha, thank you @tayvano . It's both! Made a slackbot response with the relevant information and users used it for people with questions :).

**Pull request merged**